packages/nc-gui/composables/useMultiSelect/convertCellData.ts

```diff
@@ -147,7 +151,7 @@ export default function convertCellData(
 
       const attachments = []
 
-      for (const attachment of parsedVal) {
+      for (const attachment of value ? parsedVal : files!) {
```

Comment:
I think `!` is not required. you already return null in L116 so files must not be null here.

Reply:
Yes not required but I using `&&` operator in if condition and returning null, due to that showing error in this line (L154). Thats why i used `!`

Reply:
What error did you see?

Reply:
Object is possibly 'undefined'.

To fix this I added default value as blank array 
```ts
  const { to, value, files = [], oldFiles, column } = args
```